src/ingameop.cpp

```diff
@@ -244,7 +242,7 @@ static bool _intAddInGameOptions()
 	addIGTextButton(INTINGAMEOP_OPTIONS, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_OP_W, _("Options"), OPALIGN);
 	row++;
 
-	if (!s)
+	if ((!bMultiPlayer || !NetPlay.bComms) && !bInTutorial)
```

Comment:
The original condition is easier to understand for me. 
The removal of `!= 0` is fine!

Reply:
done